test/cpp/tensorexpr/test_cuda.cpp

```diff
@@ -90,7 +90,7 @@ static void testCudaTestVectorAdd02_impl(int N, int block_size) {
       },
       [&](const VarHandle& n) { return a_buf(n) + b_buf(n); });
   Schedule sch({c});
-  VarHandle n(c->arg(0));
+  VarHandle n(c->function()->arg(0));
```

Comment:
I think a convenience wrapper that does "tensor.arg" -> "tensor.function().arg" simplifies our code.